packages/ui/src/accounts/components/AccountItem/components/lockItems/InvitationLockItem.tsx

```diff
@@ -6,18 +6,22 @@ import { useGetMemberInvitedEventsQuery } from '@/memberships/queries'
 import { LockItem } from '../LockItem'
 import { LockDetailsProps } from '../types'
 
+const tooltipLabel =
+  'Invitation lock can be spent on transaction fees and staking for proposals, voting and working groups applications. JOY tokens subject to this lock cannot be transferred to any other accounts. This lock is unrecoverable. NB: Transaction fees will first be taken from your transferable balance if it is positive.'
+
 export const InvitationLockItem = ({ lock, address, isRecoverable }: LockDetailsProps) => {
   const { data } = useGetMemberInvitedEventsQuery({ variables: { lockAccount: address } })
   const eventData = data?.memberInvitedEvents[0]
   const createdInEvent = eventData && asBlock(eventData)
+  const recoveryInfo = { unrecoverableLabel: 'Unrecoverable', tooltipLabel }
```

Comment:
A bit strange not to declare `tooltipLabel` here since it's only used once

Reply:
True, at some point i expect to implement translations then it would get factored out, see 1989.